src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -136,15 +136,14 @@ public org.folio.des.domain.dto.Job upsert(org.folio.des.domain.dto.Job jobDto,
       result.setExitStatus(ExitStatus.UNKNOWN);
     }
 
-    var jobCommand = jobExecutionService.prepareStartJobCommand(result);
-
     log.info("Upserting {}.", result);
     result = repository.save(result);
     log.info("Upserted {}.", result);
 
-    jobCommand.setId(result.getId());
-
-    jobExecutionService.sendJobCommand(jobCommand);
+    if (withJobCommandSend) {
+      var jobCommand = jobExecutionService.prepareStartJobCommand(result);
+      jobExecutionService.sendJobCommand(jobCommand);
+    }
```

Comment:
This flag was deleted by mistake in some of previous PR, that brings sending event to Kafka twice for EDIFACT orders:
https://github.com/folio-org/mod-data-export-spring/blob/master/src/main/java/org/folio/des/builder/scheduling/EdifactScheduledTaskBuilder.java#L44

Better to refactor this code to avoid flag and use new method for sending kafka records instead, but it requires a big refactoring especially in conversion between entity and model of Job class instance(that is not suitable for bug fixes)